src/components/Appbar.js

```diff
@@ -1,19 +1,60 @@
-import * as React from 'react';
-import AppBar from '@mui/material/AppBar';
-import Box from '@mui/material/Box';
-import Toolbar from '@mui/material/Toolbar';
-import Typography from '@mui/material/Typography';
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import PrimaryButton from "./PrimaryButton";
+import Grid from "@mui/material/Grid";
+import Link from "@mui/material/Link";
 export default function DenseAppBar() {
   return (
-    <Box sx={{ flexGrow: 1}}>
-      <AppBar position="static" >
-        <Toolbar variant="dense">
-          <Typography variant="h5" color="#212121" component="div" height="64px"
-          >
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar position="static">
+        <Toolbar
+          variant="dense"
+          sx={{
+            display: "flex",
+            justifyContent: "space-between",
+          }}
+        >
+          <Typography variant="h5" color="#212121">
             Company Name
           </Typography>
+          <Grid
+            sx={{
+              width: "22%",
```

Comment:
WHy do we need to give width here? any specific reason?

Reply:
it was by mistake, now using colums instead